riotbuild/Dockerfile

```diff
@@ -151,7 +151,7 @@ ENV MIPS_ELF_ROOT /opt/mips-mti-elf/${MIPS_VERSION}
 ENV PATH ${PATH}:${MIPS_ELF_ROOT}/bin
 
 # Install RISC-V binary toolchain
-ARG RISCV_VERSION=10.1.0-1.1
+ARG RISCV_VERSION=10.2.0-1.2
```

Comment:
Unrelated?

Reply:
That's the goal of this PR :)

Reply:
Ahh sorry, I didn't read the PR title, only the description.